beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -5878,6 +5876,29 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             .flatten()
     }
 
+    /// The epoch since which we cater blob data upon a request 'ByRoot'.
+    /// `None` if the `Eip4844` fork is disabled.
+    pub fn data_availability_boundary_by_root_rpc_request(&self) -> Option<Epoch> {
```

Comment:
I think it'd be better if this were named something more general (I can't think of a good name right now outside of something really obvious like `finalized_data_availability_{boundary|epoch}`), because this concept might end up being relevant outside of this one request.  also keeps networking spec out of the beacon chain